src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,15 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private int size;
+
     public Excavator() {
     }
 
+    public Excavator(int size) {
+        this.size = size;
+    }
+
```

Comment:
what about inheriting the parent's class fields? (Machine) and initializing it here?

Reply:
same for all child classes